server/cpho/models/phacorgs.py

```diff
@@ -0,0 +1,81 @@
+from django.contrib import admin
+from django.db import models
+
+from server import fields
+from server.model_util import add_to_admin
+
+# from cpho.models import User
+from cpho.text import tdt
+from cpho.util import get_lang_code
+
+
+def _get_phacorg_str(self, lang="en"):
+    if lang == "fr":
+        s = self.name_fr
+    else:
+        s = self.name_en
+    if self.parent is None or self.parent_id == self.id:
+        return s
+    return f"{s} < {_get_phacorg_str(self.parent)}"
+
+
+def _get_phacorg_acronym(self, lang="en"):
+    if lang == "fr":
+        acronym = self.acronym_fr
+    else:
+        acronym = self.acronym_en
+    if self.parent is None or self.parent_id == self.id:
+        return acronym
+    return f"{_get_phacorg_acronym(self.parent)}-{acronym}"
+
+
+@add_to_admin
+class PHACOrg(models.Model):
+    def __str__(self):
+        s = _get_phacorg_acronym(self, get_lang_code())
+        s += tdt("colon") + " "
```

Comment:
I think ": " would work just fine, no need to translate the colon, right now there's no space in between the acronym and the "colon"